actions/workflows/st2_e2e_tests.yaml

```diff
@@ -1,300 +1,199 @@
-        token: "{{st2_token}}"
-        action: "tests.test_quickstart_python_actions"
-        params: "token={{st2_token}} protocol={{protocol}}"
-        hosts: "{{hostname}}"
-        timeout: 600
-      on-success: "test_mistral_examples"
-    -
-      name: "test_mistral_examples"
-      ref: "st2cd.action_run"
-      params:
-        env:
-          ST2_BASE_URL: "{{protocol}}://{{hostname}}"
-          ST2_AUTH_URL: "{{protocol}}://{{hostname}}:9100"
-          ST2_API_URL: "{{protocol}}://{{hostname}}:9101"
-        name: "test_mistral_examples"
-        token: "{{st2_token}}"
-        params: ""
-        action: "examples.mistral_examples"
-        hosts: "{{hostname}}"
-        timeout: 600
-      on-success: "test_timer_rule"
-      on-failure: "test_timer_rule"
-    -
-      name: "test_timer_rule"
-      ref: "st2cd.action_run"
-      params:
-        env:
-          ST2_BASE_URL: "{{protocol}}://{{hostname}}"
-          ST2_AUTH_URL: "{{protocol}}://{{hostname}}:9100"
-          ST2_API_URL: "{{protocol}}://{{hostname}}:9101"
-        name: "test_timer_rule"
-        token: "{{st2_token}}"
-        action: "tests.test_timer_rule"
-        params: "token={{st2_token}} protocol={{protocol}}"
-        hosts: "{{hostname}}"
-        timeout: 600
-      on-success: "test_windows_runners"
-      on-failure: "test_windows_runners"
-    -
-      name: "test_windows_runners"
-      ref: "st2cd.action_run"
-      params:
-        env:
-          ST2_BASE_URL: "{{protocol}}://{{hostname}}"
-          ST2_AUTH_URL: "{{protocol}}://{{hostname}}:9100"
-          ST2_API_URL: "{{protocol}}://{{hostname}}:9101"
-        name: "test_windows_runners"
-        token: "{{st2_token}}"
-        action: "tests.test_windows_runners"
-        params: "token={{st2_token}} protocol={{protocol}} windows_host={{windows_host}} windows_username={{windows_username}} windows_password={{windows_password}}"
-        hosts: "{{hostname}}"
-        timeout: 600
-      on-success: "selenium"
-      on-failure: "selenium"
-    -
-      name: "selenium"
-      ref: "st2cd.action_run"
-      params:
-        env:
-          ST2_BASE_URL: "{{protocol}}://{{hostname}}"
-          ST2_AUTH_URL: "{{protocol}}://{{hostname}}:9100"
-          ST2_API_URL: "{{protocol}}://{{hostname}}:9101"
-        name: "webui_selenium"
-        token: "{{st2_token}}"
-        action: "webui.selenium"
-        params: "token={{st2_token}} protocol={{protocol}} branch=master repo=https://github.com/StackStorm/st2e2e.git web_host={{hostname}}"
-        hosts: "{{hostname}}"
-        timeout: 600
-      on-failure: "ignore_selenium_failure"
-    -
-      name: "ignore_selenium_failure"
-      ref: "core.local"
-      params:
-        cmd: "echo 'Selenium tests failed, but all other tests were successful. Setting overall status to success'"
+version: '2.0'
+name: st2cd.st2_e2e_tests
+description: Run the suite of end to end tests on the st2 host
 
-  default: "get_st2_token"
+workflows:
```

Comment:
We run them as an entire chain today anyway and there's no way to run them separately today.  Also, I addressed @dzimine concerns offline.  The tests are visible in the UI.